repo/packages/M/marathon/9/config.json

```diff
@@ -0,0 +1,378 @@
+          "type" : "integer"
+        },
+        "heap-max" : {
+          "default" : 768,
+          "description" : "Memory (MB) max size for the JVM heap. This number should be less than the memory allocated to the Marathon instance (General rule: 50%).",
+          "minimum" : 0,
+          "type" : "integer"
+        }
+      },
+      "required" : [ "heap-min", "heap-max" ],
+      "type" : "object"
+    },
+    "marathon" : {
+      "additionalProperties" : true,
+      "description" : "Marathon command line flags. These are the same flags that are passed through to Marathon when launching manually from the command line. See details here: https://mesosphere.github.io/marathon/docs/command-line-flags.html",
+      "properties" : {
+        "access-control-allow-origin" : {
+          "description" : "The origin(s) to allow in Marathon. Not set by default. Example values are \"*\", or \"http://localhost:8888, http://domain.com\"",
+          "type" : "string"
+        },
+        "artifact-store" : {
+          "description" : "URL to the artifact store. Supported store types hdfs, file. Example: hdfs://localhost:54310/path/to/store, file:///var/log/store",
+          "type" : "string"
+        },
+        "assets-path" : {
+          "description" : "Set a local file system path to load assets from, instead of loading them from the packaged jar.",
+          "type" : "string"
+        },
+        "checkpoint" : {
+          "description" : "Enabled: (Default) Enable checkpointing of tasks. Requires checkpointing enabled on slaves. Allows tasks to continue running during mesos-slave restarts and upgrades Disabled: Disable checkpointing of tasks.",
+          "type" : "boolean",
+          "default" : true
+        },
+        "decline-offer-duration" : {
+          "description" : "(Default: 120 seconds) The duration (milliseconds) for which to decline offers by default",
+          "type" : "integer",
+          "default" : 120000
+        },
+        "default-accepted-resource-roles" : {
+          "description" : "Default for the defaultAcceptedResourceRoles attribute of all app definitions as a comma-separated list of strings.This defaults to all roles for which this Marathon instance is configured to receive offers.",
+          "type" : "string"
+        },
+        "disable-http" : {
+          "description" : "Disable listening for HTTP requests completely. HTTPS is unaffected.",
+          "type" : "boolean",
+          "default" : false
+        },
+        "env-vars-prefix" : {
+          "description" : "Prefix to use for environment variables",
+          "type" : "string"
+        },
+        "event-stream-max-outstanding-messages" : {
+          "description" : "The event stream buffers events, that are not already consumed by clients. This number defines the number of events that get buffered on the server side, before messages are dropped.",
+          "type" : "integer",
+          "default" : 50
```

Comment:
Technically there is no minimum on the outstanding message count and we also do not validate it in Marathon. To remove the minimum is ok for that reason.